File: Java/BroadcastCommand.java
// Register this executor for the "broadcast" command in your plugin.yml.
// Command: /broadcast <message>
public class BroadcastCommand implements CommandExecutor {

    @Override
    public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {
        if (args.length == 0) {
            sender.sendMessage("§cUsage: /broadcast <message>");
            return true;
        }

        String message = String.join(" ", args);
        message = ChatColor.translateAlternateColorCodes('&', message);
        Bukkit.broadcastMessage("§6[§eBroadcast§6] §f" + message);
        return true;
    }
}

File: Kotlin/BroadcastCommand.kt
package org.example.pluginname.command //Package

import org.bukkit.Bukkit
import org.bukkit.ChatColor
import org.bukkit.command.Command
import org.bukkit.command.CommandExecutor
import org.bukkit.command.CommandSender

// Command: /broadcast <message>
class BroadcastCommand : CommandExecutor {
    override fun onCommand(sender: CommandSender, command: Command, label: String, args: Array<out String>): Boolean {
        if (args.isEmpty()) {
            sender.sendMessage("§cUsage: /broadcast <message>")
            return true
        }

        val message = ChatColor.translateAlternateColorCodes('&', args.joinToString(" "))
        Bukkit.broadcastMessage("§6[§eBroadcast§6] §f$message")
        return true
    }
}
